src/test/test_mempool.cc

```diff
@@ -402,6 +402,33 @@ TEST(mempool, btree_map_test)
   ASSERT_EQ(0, mempool::osd::allocated_bytes());
 }
 
+TEST(mempool, check_shard_select)
```

Comment:
This is failing https://jenkins.ceph.com/job/ceph-pull-requests/70918/consoleFull#19716391392a811ea2-3e7b-466b-84b4-d13df7e35809